src/extensions/lapack/lapack-csd.lisp

```diff
@@ -74,10 +74,10 @@
 
     (let ((v2h (conjugate (/ 1.0d0 (- (* c (conjugate u2) a4)
                                       (* s (conjugate u1) a3)))))
-          (mu1 (empty '(1 1)))
-          (mu2 (empty '(1 1)))
-          (mv1h (empty '(1 1)))
-          (mv2h (empty '(1 1))))
+          (mu1 (empty '(1 1) :type '(complex double-float)))
```

Comment:
@stylewarning I think this snuck under our radar because i) we're not using this `csd-2x2-basic` anywhere except the tests, and ii) the test for this suffers from #115